diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/HeaderCommentRuleFixTest.kt

```diff
@@ -66,4 +66,46 @@ class HeaderCommentRuleFixTest : FixTestBase(
             )))
         )
     }
+
+    @Test
+    @Tag(WarningNames.HEADER_MISSING_OR_WRONG_COPYRIGHT)
+    fun `copyright multiline`() {
+        fixAndCompare("MultilineCopyrightExample.kt", "MultilineCopyrightTest.kt",
+            listOf(RulesConfig(HEADER_MISSING_OR_WRONG_COPYRIGHT.name, true, mapOf(
+                "isCopyrightMandatory" to "true",
+                "copyrightText" to """
+                |    Copyright 2018-2020 John Doe.
+                |
+                |    Licensed under the Apache License, Version 2.0 (the "License");
+                |    you may not use this file except in compliance with the License.
+                |    You may obtain a copy of the License at
+                |
+                |        http://www.apache.org/licenses/LICENSE-2.0
+                |
+                |    Unless required by applicable law or agreed to in writing, software
+                |    distributed under the License is distributed on an "AS IS" BASIS,
+                |    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+                |    See the License for the specific language governing permissions and
+                |    limitations under the License.
+                """.trimMargin()
+            )))
+        )
+    }
+
+    @Test
+    @Tag(WarningNames.HEADER_MISSING_OR_WRONG_COPYRIGHT)
+    fun `should not trigger if copyright text have different indents`() {
+        fixAndCompare("MultilineCopyrightNotTriggerExample.kt", "MultilineCopyrightNotTriggerTest.kt",
+            listOf(RulesConfig(HEADER_MISSING_OR_WRONG_COPYRIGHT.name, true, mapOf(
+                "isCopyrightMandatory" to "true",
+                "copyrightText" to """|
+    Copyright 2018-2020 John Doe.
+    
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+""".trimMargin()
```

Comment:
Please either use `trimMargin` and place `|` at all lines, or use `trimIndent`. It's confusing in this test.